SETUP/upgrade/21/20250101_drop_marc_records.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
If I understanding the naming convention correctly, should the script be `20240501_drop_marc_records.php`?

Reply:
Yes indeed - I'll get that fixed.

Reply:
Upgrade script renamed and force-pushed to update PR.